tests/simulators/newtonian/test_newtonian.py

```diff
@@ -59,3 +59,27 @@ def test_pickle(loadLocalScenario):
     simulation = simulator.simulate(scene, maxSteps=100)
     egoPos, otherPos = simulation.result.trajectory[-1]
     assert egoPos.distanceTo(otherPos) < 1
+
+
+def test_pedestrian_movement(loadLocalScenario):
+    scenario = loadLocalScenario("pedestrian.scenic", mode2D=True)
```

Comment:
For new tests, let's inline the Scenic code into the function unless there's a specific reason why the code needs to be in another file. It's easier to understand what the test does, and you can use the `getAssetPath` fixture to get the map path in a more robust way (like in the CARLA tests).